File: julia/medium/pyramidArray.jl
function pyramidArray(num::Int)
  arr::Array{Array{Int64,1}} = []

  for i in 1:num
    push!(arr, fill(1, i))
  end

  return arr
end

println(pyramidArray(0))
println(pyramidArray(1))
println(pyramidArray(2))
println(pyramidArray(3))

File: codewars/easy/pyramidArray.ts
function pyramidArray(n: number) {
	const result: number[][] = []
	let i = 1

	while (i <= n) {
		const tempArr: number[] = new Array(i)
		tempArr.fill(1)
		result.push(tempArr)

		i += 1
	}

	return result
}

export { pyramidArray }
